lib/ansiblelint/cli.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8; -*-
+import optparse
+import os
+import sys
+
+import yaml
+
+import ansiblelint
+from ansiblelint.version import __version__
+
+
+__PATH_VARS = ['exclude_paths', 'rulesdir', ]
```

Comment:
Using a double leading underscore is usually unnecessary (and even harmful when in classes).